index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for git-init

```

Comment:
what's this file for?
In doubt, please re-run:
```console
npx dts-gen --dt --name git-init --template module
```
to scaffold proper type declaration files structure for `git-init`:
https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package